### DISCUSSION TITLE
[Spring] Proxy와 프록시패턴, 데코레이터패턴에 대해서 알아보자

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#194] [1. 개발 관련 글 모음]
KEEPER R2 프로젝트에서 @LoginMember 어노테이션을 공부하다가 AOP를 공부하게 되었는데 거기서 프록시와 다양한 패턴을 공부하게 되었습니다..!!

https://02ggang9.github.io/spring/proxy/

추석에 노션 포스팅을 깃 블로그에 이관한다고 했는데 200개 정도 글이 있어서 그냥 깃 블로그에 링크만 남기기로 결정했습니다~!

**Comment by pnudpwls1103** [comment]
유용한 글 감사합니다 :)

**Reply by 02ggang9** [reply]
안녕하세용~ 오랜만이네요..!! 읽어주셔서 감사합니다 👍👍
